admin/class-boldgrid-backup-admin-cron-log.php

```diff
@@ -136,6 +136,57 @@ public function get_log() {
 		return json_decode( $contents, true );
 	}
 
+	/**
+	 * Get our scheduled jobs.
+	 *
+	 * @since 1.17.0
+	 *
+	 * @return string The markup displaying our scheduled jobs.
+	 */
+	public function get_jobs() {
+		$jobs = get_option(
+			$this->core->jobs->option,
+			array(),
+		);
+
+		// Return a message if there are no jobs.
+		if ( empty( $jobs ) ) {
+			return '<p style="font-style:italic">' . esc_html__( 'No jobs scheduled.', 'boldgrid-backup' ) . '</p>';
+		}
+
+		// Generate table heading
+		$markup = sprintf(
+			'<table class="wp-list-table widefat striped">
+				<thead>
```

Comment:
The table HTML is missing a closing </thead> tag. This could cause rendering issues in some browsers. Make sure to properly close all HTML tags.

Reply:
The closing `</tbody></table>` line is on 185

Reply:
I might be missing something, but I believe the `</thead>` is missing.